tardis/model.py

```diff
@@ -207,7 +207,7 @@ def calculate_j_blues(self, init_detailed_j_blues=False):
             logger.info('Calculating J_blues for radiate_rates_type=detailed')
 
             self.j_blues = pd.DataFrame(
-                self.j_blue_estimators.transpose() *
+                self.j_blue_estimators *
                 self.j_blues_norm_factor.value,
```

Comment:
I don't like the way this is indented, the bottom line in the multiplication should be indented one level to the right.